sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala

```diff
@@ -139,27 +140,15 @@ class QueryExecution(val sparkSession: SparkSession, val logical: LogicalPlan) {
       }
   }
 
+  @transient
+  private lazy val sessionLocalTimeZone =
```

Comment:
`toHiveString` is for test only, we don't need to keep a `lazy val` for performance, we can just inline it.